stable/ark-server/values.yaml

```diff
@@ -0,0 +1,51 @@
+# Default values for ark-server.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+configuration:
+  # cloudprovider and bucket are always needed
+  cloudprovider: ChangeMe
+  bucket: ChangeMe
+  # region is aws only
+  region: ChangeMe
+  # apitimeout is azure only
+  apitimeout: ChangeMe
+  # credentials not needed if kube2iam is enabled
+  credentials: ChangeMe
+  kube2iam: false
+  # following are default values
+  backupSyncPeriod: 60m
+  gcSyncPeriod: 60m
+  scheduleSyncPeriod: 1m
+  restoreOnlyMode: false
+
+namespace:
+  create: true
+  name: heptio-ark
+
+image:
+  repository: gcr.io/heptio-images/ark
+  tag: v0.8.1
+  pullPolicy: IfNotPresent
+
+# this must be created manually before helm install, see README
+secret:
+  name: cloud-credentials
+
+rbac:
+  create: true
+
+serviceAccount:
+  hook:
+    create: true
+    name:
+  server:
+    create: true
+    name:
+
+podAnnotations:
+
+kubectl:
```

Comment:
Did you leave empty on purpose?

Reply:
Yes. 
Do you think it's better if i add the string ` iam.amazonaws.com/role: arn:aws:iam::<AWS_ACCOUNT_ID>:role/<HEPTIO_ARK_ROLE_NAME>`
and take those 2 vars from values.yaml ?

Reply:
I added the string, I thinkg it's better

Reply:
I think he was talking about the kubectl variable.
By the charts best practices, default values should either be enough to install and run the chart or produce a warning that lists the variables the user needs to fill.
Moreover the annotation you added is AWS specific, and not everyone run their cluster on aws (gke, aks, on-premise, ...).

Reply:
ah, I misunderstood, i'll change it, tnx. 
Regarding the annotation, It's only aws because as far as I saw in the ark docs, it can be used only with aws: https://github.com/heptio/ark/blob/v0.8.1/examples/aws/10-deployment-kube2iam.yaml

Reply:
ah, I misunderstood, i'll change it, tnx. 
Regarding the annotation, It's only aws because as far as I saw in the ark docs, it can be used only with aws: https://github.com/heptio/ark/blob/v0.8.1/examples/aws/10-deployment-kube2iam.yaml